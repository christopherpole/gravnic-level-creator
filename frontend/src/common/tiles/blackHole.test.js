import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import BlackHoleTile from './blackHole';

configure({ adapter: new Adapter() });

describe('<BlackHoleTile />', () => {
  it('Renders without exploding', () => {
    const blackHoleTile = shallow(<BlackHoleTile />);

    expect(blackHoleTile).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const blackHoleTile = shallow(<BlackHoleTile />);

    expect(toJson(blackHoleTile)).toMatchSnapshot();
  });
});
