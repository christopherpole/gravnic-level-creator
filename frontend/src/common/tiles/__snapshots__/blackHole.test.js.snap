// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlackHoleTile /> Matches the current snapshot 1`] = `
<styled.div
  shrinking={false}
/>
`;
